pkg/session_recording/recordings.jsx

```diff
@@ -255,6 +344,27 @@
                     'min-width': '300px'
                 };
 
+                let speed = (() => {
+                    if (this.state.speed === 0) {
+                        return '';
+                    }
+                    else if (this.state.speed > 0) {
+                        return 'x' + Math.pow(2, Math.abs(this.state.speed));
+                    }
+                    else if (this.state.speed < 0) {
+                        let speed = '' + Math.pow(2, Math.abs(this.state.speed));
+                        return '/' + speed;
+                    }
```

Comment:
Let's make this shorter and more uniform:

    let exp = this.state.speed_exp
    let factor = Math.pow(2, Math.abs(exp));
    if (exp > 0) {
        return 'x' + factor;
    } else if (exp < 0) {
        return '/' + factor;
    } else {
        return '';
    }